micrometer-spring-legacy/src/main/java/io/micrometer/spring/autoconfigure/export/azuremonitor/AzureMonitorProperties.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.spring.autoconfigure.export.azuremonitor;
+
+import io.micrometer.spring.autoconfigure.export.StepRegistryProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * {@link ConfigurationProperties} for configuring Azure Application Insights metrics export.
+ *
+ * @author Dhaval Doshi
+ */
+@ConfigurationProperties(prefix = "management.metrics.export.azure.application-insights")
```

Comment:
@izeye I would have definitely liked this. Infact we should do this. @SergeyKanzhelev are you fine with this change? It will just be different from our core SDK. Though there would be no backport issue here. If ApplicationInsights-SpringBoot-Starter is already present on class path then it's autoconfig will take precedence over Micrometer's and the ikey should be resolved with the property there which is `azure.application-insights.instrumentation-key` . Let me know.